ait/dsn/plugins/AOS_to_CCSDS.py

```diff
@@ -40,7 +40,7 @@ def process_m_pdu_tuple(self, input_tuple):
         while remaining_bytes_to_send is not None:
             #check for empty e0 bytes
             #TODO handle the case where there are fewer than 3 idle bytes remaining
```

Comment:
@kmarwah With this updated, can the TODO comment now be removed?

Reply:
Good point. Just updated the comment.